release_notes_generator/builder.py

```diff
@@ -29,7 +29,6 @@
 logger = logging.getLogger(__name__)
 
 
-# TODO - reduce to function only after implementing the features. Will be supported more build ways?
```

Comment:
Can you please explain the conclusion of this TODO?

Reply:
I do not have this TODO in my latest code.

Reply:
Yes, the question was about, why did you remove it. It is not relevant anymore, or implemented?

Reply:
It was no more relevant.